src/libsystemd/sd-bus/bus-internal.h

```diff
@@ -321,6 +322,9 @@ struct sd_bus {
 
         /* zero means use value specified by $SYSTEMD_BUS_TIMEOUT= environment variable or built-in default */
         usec_t method_call_timeout;
+
+        /* See static uint8_t bus_object_nonce[] in sd-bus.c */
+        uint8_t bus_object_nonce[BUS_OBJECT_NONCE_SIZE];
```

Comment:
16 byte sounds a bit like overkill, we just want to detect accidental programming mistakes, not avoid collisions at all costs..